config-store-yaml/src/test/java/io/stargate/config/store/api/yaml/ConfigStoreActivatorYamlDoesNotExistsTest.java

```diff
@@ -0,0 +1,49 @@
+/*
+ * Copyright The Stargate Authors
+ *
+ * Licensed under the Apache License, Version 2.0 (the "License");
+ * you may not use this file except in compliance with the License.
+ * You may obtain a copy of the License at
+ *
+ * http://www.apache.org/licenses/LICENSE-2.0
+ *
+ * Unless required by applicable law or agreed to in writing, software
+ * distributed under the License is distributed on an "AS IS" BASIS,
+ * WITHOUT WARRANTIES OR CONDITIONS OF ANY KIND, either express or implied.
+ * See the License for the specific language governing permissions and
+ * limitations under the License.
+ */
+package io.stargate.config.store.api.yaml;
+
+import static org.mockito.ArgumentMatchers.any;
+import static org.mockito.ArgumentMatchers.eq;
+import static org.mockito.Mockito.mock;
+import static org.mockito.Mockito.times;
+import static org.mockito.Mockito.verify;
+
+import io.stargate.config.store.api.ConfigStore;
+import org.junit.jupiter.api.BeforeAll;
+import org.junit.jupiter.api.Test;
+import org.osgi.framework.BundleContext;
+
+class ConfigStoreActivatorYamlDoesNotExistsTest {
+
+  @BeforeAll
+  public static void setup() {
+    System.setProperty("stargate.config_store.yaml.location", "non_existing_path");
```

Comment:
This will affect of the other unit tests in the same JVM, I think... could lead to odd failures.

Reply:
Why not add another constructor to allow unit tests to pass specific settings? The default constructor would use system properties... WDYT?

Reply:
good idea, done.